hibernate-core/src/test/java/org/hibernate/orm/test/annotations/id/JoinColumnOverrideTest.java

```diff
@@ -33,10 +33,10 @@
 @BaseUnitTest
 public class JoinColumnOverrideTest {
 
-	private static final String expectedSqlPointyTooth = "create table PointyTooth (id numeric(128,0) not null, " +
-			"bunny_id numeric(128,0), primary key (id))";
-	private static final String expectedSqlTwinkleToes = "create table TwinkleToes (id numeric(128,0) not null, " +
-			"bunny_id numeric(128,0), primary key (id))";
+	private static final String expectedSqlPointyTooth = "create table PointyTooth (bunny_id numeric(128,0), " +
+			"id numeric(128,0) not null, primary key (id))";
+	private static final String expectedSqlTwinkleToes = "create table TwinkleToes (bunny_id numeric(128,0), " +
+			"id numeric(128,0) not null, primary key (id))";
```

Comment:
@beikov why is this impacting the `not null` constraint?

Reply:
It just reordering the columns. The diff looks confusing.

Reply:
Ah, OK, sorry.